src/_actions/assetType.action.js

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+
+import constants from '../_constants';
+// import apiConfig from '../config';
+
+const { LOAD_ASSET_TYPE_SUCCESS, LOAD_ASSET_TYPE_FAILURE } = constants;
+
+// const url = apiConfig[process.env.NODE_ENV];
+
+export const loadAssetTypeAction = (page, limit = 10) => {
+  return (dispatch) => {
+    axios.get(`http://localhost:3004/assetTypes?_page=${page}&_limit=${limit}`)
```

Comment:
Is using `localhost` for API calls going to be permanent?

Reply:
I see that we already have an `apiConfig` object with URLs defined for the various dev environments. I think we should use that instead to make this more flexible

Reply:
I have fixed it. The was changing the URL I configured that was why I did that to bypass the error but failed to removed when I finished.